### ISSUE TITLE
Support `exit_group` syscall (#231)

### ISSUE THREAD

**Issue by novafacing** [trailofbits/maat#88]
The emulator prints out:
```
[Error] Exception in CALLOTHER handler: SYSCALL: EnvEmulator: syscall '231' not supported for emulation
[Error] Unexpected error when processing IR instruction, aborting...
```
upon executing an `exit_group` syscall, which is somewhat opaque given it happens quite frequently and is not really an error. Is there a nicer way we could implement `exit`-like syscalls to make things more clear to the user that execution has ended (possibly with the exit code) instead of showing an error?

**Comment by Boyan-MILANOV** [comment]
Yeah we just never took the time to implement the syscall handlers, but there is already a `MaatEngine::terminate_process()` method for exiting the symbolically executed process. I'll make a PR to support `exit` and `exit_group`.

**Comment by Boyan-MILANOV** [comment]
Implemented `exit` and `exit_group` in #89
